pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -23,15 +23,24 @@
 import pytorch_lightning as pl
 from pytorch_lightning.callbacks.base import Callback
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _PSUTIL_AVAILABLE
+from pytorch_lightning.utilities.memory import get_cpu_process_metrics
+from pytorch_lightning.utilities.rank_zero import rank_zero_deprecation, rank_zero_warn
 from pytorch_lightning.utilities.types import STEP_OUTPUT
-from pytorch_lightning.utilities.warnings import rank_zero_deprecation
 
 
 class DeviceStatsMonitor(Callback):
     r"""
     Automatically monitors and logs device stats during training stage. ``DeviceStatsMonitor``
     is a special callback as it requires a ``logger`` to passed as argument to the ``Trainer``.
 
+    Args:
+        cpu_stats: if ``None``, it will log CPU stats only if the accelerator is CPU.
+            It will raise a warning if `psutil` is not installed till version 1.9.0.
+            if ``True``, it will log CPU stats regardless of the accelerator, and it will
+            raise an exception if `psutil` is not installed.
+            if ``False``, it will not log CPU stats regardless of the accelerator.
```

Comment:
```suggestion
        cpu_stats: if ``None``, it will log CPU stats only if the accelerator is CPU.
            It will raise a warning if `psutil` is not installed till version 1.9.0.
            If ``True``, it will log CPU stats regardless of the accelerator, and it will
            raise an exception if `psutil` is not installed.
            If ``False``, it will not log CPU stats regardless of the accelerator.
```